### DISCUSSION TITLE
Why does it  take so much time for my fullnode to use some apis

### DISCUSSION THREAD

**Discussion by panchengj** [filecoin-project/lotus#9522] [Build and Sync]
Why does it  take so much time for my fullnode to use some apis
for example:
![image](https://user-images.githubusercontent.com/13955506/196852209-820d4fa6-5d0e-4692-909f-7305c385e6a9.png)

**Comment by panchengj** [comment]
And  server resource consumption is very low

**Comment by panchengj** [comment]
the block heights seems far from the latest height, i increase LOTUS_CHAIN_INDEX_CACHE to  65536, but it seems not work after restarting  lotus

**Comment by TippyFlitsUK** [comment]
Can you elaborate on the other API calls you are seeing issues with @panchengj?
Many thanks :pray:

**Reply by panchengj** [reply]
Filecoin.StateGetReceipt  
Filecoin.StateReplay
Filecoin.StateSearchMsg
they all took hundreds of seconds，how could i solve the problem? @TippyFlitsUK
